src/txn/txn_rollback_to_stable.c

```diff
@@ -476,11 +469,13 @@ __rollback_abort_newer_row_leaf(
             __rollback_abort_newer_insert(session, insert, rollback_timestamp);
 
         /* Abort any on-disk value. */
```

Comment:
Maybe update the comment to start with "If the configuration is not in-memory ..." ?

Reply:
Fixed.